core/src/main/java/io/undertow/server/handlers/PathTemplateRouterHandler.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.undertow.server.handlers;
+
+import io.undertow.server.HttpHandler;
+import io.undertow.server.HttpServerExchange;
+import io.undertow.util.PathTemplateRouter;
+import java.util.Map;
+import java.util.Objects;
+
+import static io.undertow.server.handlers.PathTemplateHandler.PATH_TEMPLATE_MATCH;
+
+/**
+ * A handler that matches URI templates.
+ *
+ * @author Dirk Roets dirkroets@gmail.com
+ * @since 2023-07-20
```

Comment:
Not sure if we are supposed to be so pedantic to require bump on this as well. @fl4via  will know.

Reply:
@dirkroets  @baranowb because it will be merged in 2024, I think we should either bump it or just omit it, keeping in mind that we are going to have 2024 in the copyright header above.

Reply:
@fl4via @baranowb , I will remove it from the new classes.  The _since_ tags are only there because my IDE adds it for some other projects that I'm working on.